### PR TITLE
Add CRD validations for wrong names of properties

### PR DESCRIPTION
**Pull request by Beastlex**
## Description
CRDs couldn't have fields like x-examples, examples only example despite of openapi specs

## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [ ] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: testing
type: feature
summary: Add validation test for wrong property names.
impact_level: low
```

### CODE REVIEW

**Review by yalosev — CHANGES_REQUESTED** [review]
Could you add a test, please

**Review by yalosev — CHANGES_REQUESTED** [review]
So, we have some failed CRDs in validation CI.
They should be fixed

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @yalosev. I've started the workflow for label 'e2e/run/yandex-cloud'...

**Review by yalosev — DISMISSED** [review]

File: candi/cloud-providers/gcp/openapi/instance_class.yaml [hunk 1]
```diff
@@ -106,7 +106,7 @@ spec:
                     Additional labels.
 
                     [More info...](https://cloud.google.com/resource-manager/docs/creating-managing-labels)
-                  x-doc-example: |
+                  x-doc-x-doc-examples: |
```
Comment (yalosev) [inline candi/cloud-providers/gcp/openapi/instance_class.yaml:109]:
```suggestion
                  x-doc-examples: |
```

File: candi/cloud-providers/yandex/openapi/instance_class.yaml [hunk 2]
```diff
@@ -114,7 +114,7 @@ spec:
                   type: object
                   description: |
                     Additional labels.
-                  x-doc-example: |
+                  x-doc-x-doc-examples: |
```
Comment (yalosev) [inline candi/cloud-providers/yandex/openapi/instance_class.yaml:117]:
```suggestion
                  x-doc-examples: |
```

File: candi/cloud-providers/yandex/openapi/instance_class.yaml [hunk 3]
```diff
@@ -233,7 +233,7 @@ spec:
...
```
Comment (yalosev) [inline candi/cloud-providers/yandex/openapi/instance_class.yaml:236]:
```suggestion
                  x-doc-examples: |
```

**Review by yalosev — DISMISSED** [review]

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/stage'...

**Pull request merged**